app/views/unattended/provisioning_templates/iPXE/ipxe_intermediate_script.erb

```diff
@@ -11,7 +11,7 @@ snippet: false
 :net<%= i %>
 isset ${net<%= i -%>/mac} || goto no_nic
 dhcp net<%= i -%> || goto net<%= i+1 %>
-chain <%= foreman_url('iPXE', mac: "${net#{i}/mac}") %> || goto net<%= i+1 %>
+chain <%= foreman_url('iPXE', {}, { mac: "${net#{i}/mac}" }) %> || goto net<%= i+1 %>
```

Comment:
I suggested `unescaped_params: {}` in the `foreman_url` method so you could do this:
```suggestion
chain <%= foreman_url('iPXE', unescaped_params: { mac: "${net#{i}/mac}" }) %> || goto net<%= i+1 %>
```